docker/tools/score-client-demo

```diff
@@ -6,4 +6,4 @@ BASH_SCRIPT_DIR=$( dirname  "${BASH_SCRIPT}")
 DOCKERFILE_NAME=Dockerfile \
 	docker-compose \
 	-f ${BASH_SCRIPT_DIR}/../../docker-compose.yml \
-	run --rm score-client bin/score-client $@
+	run --rm score-client score-client $@
```

Comment:
bit of a chicken-and-egg problem here, which is why this command does not work. The docker container for score-client doesnt have the executable on the class path yet. that fix is in this pr: https://github.com/overture-stack/score/pull/215

for consistency sake, change this line back to
```bash
run --rm score-client bin/score-client $@
```

and in a new PR, we can update the version of the score-client/score-server docker images and then use the executable name only